src/blindpsbt.cpp

```diff
@@ -98,6 +98,23 @@ bool CreateValueRangeProof(std::vector<unsigned char>& rangeproof, const uint256
     return (res == 1);
 }
 
+// Create an explicit value rangeproof which proves that the commitment commits to an explicit value
+bool CreateBlindValueProof(std::vector<unsigned char>& rangeproof, const uint256& value_blinder, const CAmount amount, const secp256k1_pedersen_commitment& value_commit, const secp256k1_generator& gen)
+{
+    // Prep rangeproof
+    size_t rangeproof_len = 5134;
```

Comment:
This should be reduced (I will find the right value and revisit this comment).

Similarly it would be good to assert the value of this variable after the proof is created.